mtp_api/apps/prison/fixtures/test_prisons.json

```diff
@@ -0,0 +1,20 @@
+[
+{
+    "model": "prison.prison",
+    "fields": {
+        "created": "2015-06-17T13:47:18.099Z",
+        "modified": "2015-06-17T13:47:18.100Z",
+        "name": "HM Brixton"
+    },
+    "pk": "BXI"
+},
+{
+    "model": "prison.prison",
+    "fields": {
+        "created": "2015-06-17T13:47:31.283Z",
+        "modified": "2015-06-17T13:47:31.283Z",
+        "name": "HM Preston"
```

Comment:
Technically, it's “HMP” (i.e. “Her Majesty’s Prison”), not “HM”, but we can drop that bit I think? Also, we should probably use obviously-fake prison names for test data to avoid confusion in case it should ever go live.